tests/nl/light_HassTurnOff.yaml

```diff
@@ -19,6 +19,7 @@ tests:
       - "Alle keukenverlichting uit"
       - "In keuken lampen uit"
       - "Lampen in keuken uit"
+      - "Kunt u de keukenverlichting uit doen?"
```

Comment:
turning on/off all lights in an area needs either `alle` or `in` before `<area>` because it otherwise can conflict with toggling an entity.